docs/modules/operation/pages/performance-data-collection/snmp-property-extenders/pointer-like-index.adoc

```diff
@@ -0,0 +1,119 @@
+
+== Configure the Pointer-Like Index Property Extender
+
+The Pointer-Like Index property extender expects three parameters, all of which are required:
+
+.Pointer-Like Index Property Extender Parameters
+[options="header", cols="1,3"]
+|===
+| Name
+| Description
+
+| source-type
+| The name of the `resourceType` associated with the source MIB table.
+
+| source-attribute
+| The alias name of the string property to import from the source MIB table.
+
+| target-index-pointer-column
+| The alias name of the column in the target MIB table whose value may be used as an index into the source MIB table.
+|===
+
+This example shows how to use `cpmCPUTotalPhysicalIndex` as a pointer-like index into the `entPhysicalTable`.
+The target resource gains a pair of string properties, which we will call `cpmCPUTotalName` and `cpmCPUTotalDescr`.
+
+[source, xml]
+----
+<resourceType name="entPhysicalEntry" label="Physical Entity" resourceLabel="${entPhysicalName} (${entPhysicalDescr}))">
+   <persistenceSelectorStrategy class="org.opennms.netmgt.collection.support.PersistAllSelectorStrategy"/>
+   <storageStrategy class="org.opennms.netmgt.collection.support.IndexStorageStrategy"/>
+</resourceType>
+
+<resourceType name="cpmCPUTotalEntry" label="Cisco CPU Total" resourceLabel="${cpmCPUTotalName} (${cpmCPUTotalDescr})">
+   <persistenceSelectorStrategy class="org.opennms.netmgt.collection.support.PersistAllSelectorStrategy" />
+   <storageStrategy class="org.opennms.netmgt.collection.support.IndexStorageStrategy" />
+</resourceType>
+
+<groups>
+  <group name="entity-physical-table" ifType="all">
+    <mibObj oid=".1.3.6.1.2.1.47.1.1.1.1.2" instance="entPhysicalEntry" alias="entPhysicalDescr" type="string"/> <1>
+    <mibObj oid=".1.3.6.1.2.1.47.1.1.1.1.7" instance="entPhysicalEntry" alias="entPhysicalName" type="string"/> <2>
+  </group>
+
+  <group name="cpm-cpu-total" ifType="all">
+    <mibObj oid=".1.3.6.1.4.1.9.9.109.1.1.1.1.2"  instance="cpmCPUTotalEntry" alias="cpmCPUTotalPhysicalIndex"  type="string" /> <3>
+    <mibObj oid=".1.3.6.1.4.1.9.9.109.1.1.1.1.8"  instance="cpmCPUTotalEntry" alias="cpmCPUTotal5minRev"        type="gauge" />
+    <property instance="cpmCPUTotalEntry" alias="cpmCPUTotalName" class-name="org.opennms.netmgt.collectd.PointerLikeIndexPropertyExtender"> <4>
+      <parameter key="source-type" value="entPhysicalEntry"/>
+      <parameter key="source-attribute" value="entPhysicalName"/> <5>
+      <parameter key="target-index-pointer-column" value="cpmCPUTotalPhysicalIndex"/>
+    </property>
+    <property instance="cpmCPUTotalEntry" alias="cpmCPUTotalDescr" class-name="org.opennms.netmgt.collectd.PointerLikeIndexPropertyExtender"> <6>
+      <parameter key="source-type" value="entPhysicalEntry"/>
+      <parameter key="source-attribute" value="entPhysicalDescr"/> <7>
+      <parameter key="target-index-pointer-column" value="cpmCPUTotalPhysicalIndex"/>
+    </property>
+  </group>
+</groups>
```

Comment:
```suggestion
<resourceType name="entPhysicalEntry" label="Physical Entity" resourceLabel="${entPhysicalName} (${entPhysicalDescr}))">
   <persistenceSelectorStrategy class="org.opennms.netmgt.collection.support.PersistAllSelectorStrategy"/>
   <storageStrategy class="org.opennms.netmgt.collection.support.IndexStorageStrategy"/>
</resourceType>

<resourceType name="cpmCPUTotalEntry" label="Cisco CPU Total" resourceLabel="${cpmCPUTotalName} (${cpmCPUTotalDescr})">
   <persistenceSelectorStrategy class="org.opennms.netmgt.collection.support.PersistAllSelectorStrategy" />
   <storageStrategy class="org.opennms.netmgt.collection.support.IndexStorageStrategy" />
</resourceType>

<groups>
  <group name="entity-physical-table" ifType="all">
    <mibObj oid=".1.3.6.1.2.1.47.1.1.1.1.2" instance="entPhysicalEntry" alias="entPhysicalDescr" type="string"/> <1>
    <mibObj oid=".1.3.6.1.2.1.47.1.1.1.1.7" instance="entPhysicalEntry" alias="entPhysicalName" type="string"/> <2>
  </group>

  <group name="cpm-cpu-total" ifType="all">
    <mibObj oid=".1.3.6.1.4.1.9.9.109.1.1.1.1.2"  instance="cpmCPUTotalEntry" alias="cpmCPUTotalPhysicalIndex"  type="string" /> <3>
    <mibObj oid=".1.3.6.1.4.1.9.9.109.1.1.1.1.8"  instance="cpmCPUTotalEntry" alias="cpmCPUTotal5minRev"        type="gauge" />
    <property instance="cpmCPUTotalEntry" alias="cpmCPUTotalName" class-name="org.opennms.netmgt.collectd.PointerLikeIndexPropertyExtender"> <4>
      <parameter key="source-type" value="entPhysicalEntry"/>
      <parameter key="source-attribute" value="entPhysicalName"/> <5>
      <parameter key="target-index-pointer-column" value="cpmCPUTotalPhysicalIndex"/>
    </property>
    <property instance="cpmCPUTotalEntry" alias="cpmCPUTotalDescr" class-name="org.opennms.netmgt.collectd.PointerLikeIndexPropertyExtender"> <6>
      <parameter key="source-type" value="entPhysicalEntry"/>
      <parameter key="source-attribute" value="entPhysicalDescr"/> <7>
      <parameter key="target-index-pointer-column" value="cpmCPUTotalPhysicalIndex"/>
    </property>
  </group>
</groups>
```

Reply:
@mmahacek What's the change here?
GH does not like it:
![image](https://user-images.githubusercontent.com/4995222/150863456-571592bd-6ce7-45ad-88e5-318be92f1023.png)

Reply:
Sorry, @mfuhrmann, I had copied the wrong block from my local system.  The numbers need to be adjusted to line up with the updated footnotes.